renderdoc/driver/gl/gl_hooks_egl.cpp

```diff
@@ -509,6 +505,12 @@ __attribute__((visibility("default"))) EGLBoolean eglSwapBuffers(EGLDisplay dpy,
   eglhooks.eglQuerySurface_real(dpy, surface, EGL_HEIGHT, &height);
   eglhooks.eglQuerySurface_real(dpy, surface, EGL_WIDTH, &width);
 
+  GLInitParams &init = eglhooks.GetDriver()->GetInitParams();
+  int colorspace;
+  eglhooks.eglQuerySurface_real(dpy, surface, EGL_COLORSPACE, &colorspace);
+  // GL_SRGB8_ALPHA8 is specified as color-renderable, unlike GL_SRGB8.
+  init.isSRGB = init.colorBits == 32 && colorspace == EGL_COLORSPACE_sRGB;
```

Comment:
Right, but GL has copied these values for interchangeability.

If EGL_KHR_gl_colorspace is not supported, then the query would leave the return value unmodified on failure. Are you satisfied if I initialize the colorspace value to 0 so we will do the safe thing of setting isSRGB=0 ?